### PR TITLE
NIFI-1518 InferAvroSchema note has an option to set CSV delimiter

### PR DESCRIPTION
**Pull request by michalklempa**
Code transliterated from ConvertCSVToAvro processor, new Property DELIMITER (default ',') added.

### CODE REVIEW

**Comment by trkurc** [comment]
reviewing this.

File: nifi-nar-bundles/nifi-kite-bundle/nifi-kite-processors/src/test/java/org/apache/nifi/processors/kite/TestInferAvroSchema.java [hunk 1]
```diff
@@ -51,6 +51,7 @@ public void setup() {
         runner.setProperty(InferAvroSchema.GET_CSV_HEADER_DEFINITION_FROM_INPUT, "true");
         runner.setProperty(InferAvroSchema.SCHEMA_DESTINATION, InferAvroSchema.DESTINATION_CONTENT);
         runner.setProperty(InferAvroSchema.HEADER_LINE_SKIP_COUNT, "0");
+        runner.setProperty(InferAvroSchema.DELIMITER, ",");
```
Comment (trkurc) [inline nifi-nar-bundles/nifi-kite-bundle/nifi-kite-processors/src/test/java/org/apache/nifi/processors/kite/TestInferAvroSchema.java]: do you object to leaving this line out to ensure correct "default" behavior?

Comment (michalklempa) [inline nifi-nar-bundles/nifi-kite-bundle/nifi-kite-processors/src/test/java/org/apache/nifi/processors/kite/TestInferAvroSchema.java]: No.

**Comment by JPercivall** [comment]
@trkurc what was the result of your review?

**Comment by trkurc** [comment]
@JPercivall - code looks good, just need to contrib-check, do some quick manual testing and merge

**Pull request closed without merging**